services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.js

```diff
@@ -33,6 +33,19 @@ const CatalogMultiselectFilter = ({
   const triggerRef = useRef()
   const contentRef = useRef()
 
+  useEffect(() => {
+    const handleEsc = (event) => {
+      if (event.keyCode === 27) {
+        setOpen(false)
+      }
+    }
+    window.addEventListener('keydown', handleEsc)
```

Comment:
is it possible to use `useEventListener` here? see line 41: https://github.com/carbon-design-system/carbon-platform/blob/42ac07e96f91671adcaa66746cdc51c447b634c8/services/web-app/components/catalog-sort/catalog-sort.js#L41

Reply:
Oo I'm moving this to it's own function.  let me know what you think